MainModule/Server/Commands/Moderators.lua

```diff
@@ -3450,31 +3450,37 @@ return function(Vargs, env)
 			AdminLevel = "Moderators";
 			Function = function(plr,args)
 				for i,v in pairs(service.GetPlayers(plr,args[1])) do
-					local pos = v.Character.HumanoidRootPart.CFrame
+					local pos = v.Character and (v.Character:FindFirstChild('HumanoidRootPart') or (v:FindFirstChildWhichIsA("Humanoid") and v:FindFirstChildWhichIsA("Humanoid").RootPart))
+					assert(pos,'RootPart could not be found')
+					pos = pos.CFrame
```

Comment:
Could be written more efficiently as
`
local pos = assert(v.Character and (v.Character:FindFirstChild('HumanoidRootPart') or (v:FindFirstChildWhichIsA("Humanoid") and v:FindFirstChildWhichIsA("Humanoid").RootPart)), "RootPart could not be found")
`.CFrame

Reply:
doesn't really matter though